.github/PULL_REQUEST_TEMPLATE/Improvement.md

```diff
@@ -0,0 +1,19 @@
+---
+name: ⚙ Improvement
+about: Do you have some improvement to make arkitect better? 🎁
+---
+
+### Improvement
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| New Feature | yes
+| RFC         | yes/no
```

Comment:
I don't think we have RFC, right? I think we can remove this line..

Reply:
I agree with you, I remove it.